node/components/executor/src/lib.rs

```diff
@@ -39,6 +39,8 @@ pub struct Config {
     pub public_addr: net::Host,
     /// Maximal size of the block payload.
     pub max_payload_size: usize,
+    /// The duration of the view timeout, in milliseconds.
+    pub view_timeout: usize,
```

Comment:
why removing strong typing?, use time::Duration

Reply:
I thought we didn't have Duration in Proto. I'll fix it.